rpd_generator/bdl_structure/bdl_commands/system.py

```diff
@@ -1330,6 +1394,73 @@ def populate_air_energy_recovery(self):
                 self.air_energy_recovery_outdoor_airflow
             )
 
+    def populate_fan_operation_during_occupied(self):
+        fan_sch_name = self.keyword_value_pairs.get(BDL_SystemKeywords.FAN_SCHEDULE)
+
+        if (
+            self.keyword_value_pairs.get(BDL_SystemKeywords.TYPE)
+            == BDL_SystemTypes.RESVVT
+        ):
+            return FanSystemOperationOptions.CYCLING
+
+        elif (
+            self.keyword_value_pairs.get(BDL_SystemKeywords.TYPE)
+            == BDL_SystemTypes.DOAS
+        ):
+            fan_sch = self.rmd.bdl_obj_instances.get(fan_sch_name)
+            if not fan_sch:
+                return FanSystemOperationOptions.INTERMITTENT
```

Comment:
I am not sure this is definitively intermittent because "the DOAS fans operate whenever one or more of the served systems has a fan schedule value of 1" so it would depend on the fan schedules associated with the systems that the DOAS serves.

Reply:
That's a good point, the DOAS could be continuous if even one of the systems downstream is continuous

Reply:
I am thinking about this differently now where -999 on the DOAS fan schedule essentially means "I (modeler) don't want to say when occupied/unoccupied is on my DOAS. I'm going to let the simulation decide when occupied is and turn on the DOAS fan when it is occupied." so this would in effect mean continuous operation during occupied but the boundaries of occupied vs unoccupied are not defined.

I think the only curveball for this is if a fan schedule has both 1's and -999's in it. A "1" would seemingly indicate an occupied hour, however it is less definitive because the DOAS can actually be off during "1" hours if all systems-served are scheduled off and DOAS is attached at the mixed air section.

Reply:
Quick Return Cases
No DOAS fan schedule - CONTINUOUS
DOAS fan schedule is all -999 - CONTINUOUS

What are the DOAS occupied hours?
- Any system served has fan schedule with 1 at that hour
- DOAS fan schedule has a 1 at that hour

What is the DOAS operation during these hours?
- Is DOAS fan schedule 0 or -1 at all occupied hours? 	- return KEEP_OFF
- Is DOAS fan schedule 0 or -1 at some occupied hours?	- return INTERMITTENT
- Are all zones attached at the mixed air section AND are there any times where all systems-served fan schedules are 0 while DOAS fan schedule is 1?  
	- return INTERMITTENT

- ELSE: Is DOAS fan schedule 1 or -999 at all occupied hours?
	- return CONTINUOUS